src/PKSim.Core/Repositories/IParametersInContainerRepository.cs

```diff
@@ -1,12 +1,10 @@
 ﻿using OSPSuite.Core.Domain;
-using OSPSuite.Utility.Collections;
 using PKSim.Core.Model;
 
 namespace PKSim.Core.Repositories
 {
-   public interface IParametersInContainerRepository : IStartableRepository<ParameterMetaData>
+   public interface IParametersInContainerRepository
    {
```

Comment:
why it should not be startable anymore?

Reply:
As mentioned in previous comment, this whole repository does not work. It is actually more a service than a repository because it needs to retrieve, for a live parameter `parameter` the corresponding `ParameterMetaDatra`. 


The discussed implementation by path simply cannot work.

Reply:
I think that an easy way to achieve what we want is to save the ValueOriginId from the database (integer, costs nothing) in the ValueOrigin. This will not be used in MoBi but will allow PK-Sim to grab the original value origin from the database at any time independent from parameter path, formulat type etc..
At any rate this has nothing to do with this PR.